guides/doc-Provisioning_Guide/topics/Virt-VMWare.adoc

```diff
@@ -194,9 +194,14 @@ To create a host for a VMware vSphere server, complete the following steps:
 . Verify that the *Managed*, *Primary*, and *Provision* options are automatically selected for the first interface on the host. If not, select them.
 . In the interface window, review the VMware vSphere-specific fields that are populated with settings from our compute profile. Modify these settings to suit your needs.
 . Click the *Operating System* tab, and confirm that all fields automatically contain values.
-. For network-based provisioning, ensure that the *Provisioning Method* is set to `Network Based`. For image-based provisioning, ensure that the *Provisioning Method* is set to `Image Based`
+. Select the Provisioning Method that you want:
++
+* For network-based provisioning, click *Network Based*.
+* For image-based provisioning, click *Image Based*.
+* For boot-disk provisioning, click *Bood disk based*.
```

Comment:
Can you check the UI for *Boot-disk based* just to verify? Ideally, it should be hyphenized and ideally capitalized the same way as others.